src/main/java/ua/ithillel/lesson23/HeroService.java

```diff
@@ -0,0 +1,31 @@
+package ua.ithillel.lesson23;
+
+import ua.ithillel.lesson22.Hero;
+import ua.ithillel.lesson22.HeroDao;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class HeroService {
+    private final HeroDao heroDao;
+    private final HeroMovieService heroMovieService;
+
+    public HeroService(HeroDao heroDao, HeroMovieService heroMovieService) {
+        this.heroDao = heroDao;
+        this.heroMovieService = heroMovieService;
+    }
+
+    public List<HeroDto> getHeroes() {
+        List<Hero> heroes = heroDao.findAll();
```

Comment:
цей метод можна написати в один return: 
```suggestion
        return heroDao.findAll().stream().....;
```
виправиш?

Reply:
Привет, да, поправлю